x-pack/plugins/alerting/common/rule.ts

```diff
@@ -166,7 +183,20 @@ export interface Rule<Params extends RuleTypeParams = never> {
   viewInAppRelativeUrl?: string;
 }
 
-export type SanitizedRule<Params extends RuleTypeParams = never> = Omit<Rule<Params>, 'apiKey'>;
+export type SanitizedRuleAction = Omit<RuleAction, 'alertsFilter'> & {
+  alertsFilter?: {
```

Comment:
nit: maybe we could define a `SanitizedAlertsFilter` type that is built off of the `AlertsFilter` type so we can see how those are related? With these two `alertsFilter` definitions, I could see us adding a new field to the `AlertsFilter` interface and forgetting to add it here or vice versa.

Reply:
done